gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -321,7 +327,13 @@ verify_model_path (const GstTensorFilterPrivate * priv)
   if (g_strcmp0 (prop->fwname, "custom-easy") == 0)
     return TRUE;
 
-  if ((prop->model_files != NULL) && (priv->fw->verify_model_path == TRUE)) {
+  if (GST_TF_FW_V0 (priv->fw)) {
+    verify_model_path = priv->fw->verify_model_path;
+  } else {
+    verify_model_path = priv->info.verify_model_path;
```

Comment:
This is not very extensible approach.
do:
```
else if {GST_TF_FW_V1 (priv->fw)) {
...
} else {
  FAIL: NOT SUPPORTED API Version.
}
```

Reply:
Will update to 
```
else if {GST_TF_FW_V1 (priv->fw)) {
...
}
```

Is the else case needed. More details on this in https://github.com/nnsuite/nnstreamer/pull/2130#discussion_r389509768